lib/Makefile

```diff
@@ -40,7 +40,7 @@ LIBVER=$(LIBVER_MAJOR).$(LIBVER_MINOR).$(LIBVER_PATCH)
 DESTDIR?=
 PREFIX ?= /usr/local
 CFLAGS ?= -O3
-CFLAGS += -I. -std=c99 -Wall -Wextra -Wundef -Wshadow -Wcast-align -Wcast-qual -Wstrict-prototypes -pedantic
+CFLAGS += -I. -std=gnu99 -Wall -Wextra -Wundef -Wshadow -Wcast-align -Wcast-qual -Wstrict-prototypes -pedantic
```

Comment:
Part of the issue is that djgpp has a (semi-standard) _DOS_ API in addition to a _POSIX_ (almost-)compliant API.  `-std=gnu99` works around an issue in where `__STRICT_ANSI__` is defined with `-std=c99` causing DJ's headers to essentially nullify [`io.h`][0], [`fcntl.h`][1], and [`unistd.h`][2].  Essentially, it seems like a bug in djgpp's headers. The djgpp maintainers are currently discussing pushing djgpp 2.05 with gcc 5.1.0, so it is probably a good time to get such bugs fixed.

I can test the change on OSX 10.10 and make sure there are no issues in compiling or basic CLU usage with `clang`.  Beyond that, I'm open to your suggestions.  I was able to do a brief check of `clang -std=gnu99 -E dM - </dev/null` (dump preprocessor defines) and noted that it seemed to undefine `__STRICT_ANSI__` as well.  I assume `-std=gnu99` is a _C99+_ mode [(3)][3][(4)][4][(5)][5], per the docs.  

[0]: http://www.delorie.com/bin/cvsweb.cgi/djgpp/include/io.h?rev=1.8
[1]: http://www.delorie.com/bin/cvsweb.cgi/djgpp/include/fcntl.h?rev=1.11
[2]: http://www.delorie.com/bin/cvsweb.cgi/djgpp/include/unistd.h?rev=1.21
[3]: https://gcc.gnu.org/onlinedocs/gcc-4.9.2/gcc/Standards.html
[4]: https://gcc.gnu.org/c99status.html
[5]: https://gcc.gnu.org/onlinedocs/gcc-4.9.2/gcc/C-Extensions.html#C-Extensions